fakepkg

```diff
@@ -55,19 +56,15 @@ function fakebuild() {
 
 	# Assembling a list of all files belonging to the package
 	# The tar command is unable to handle very many arguments therefore create a file
-	pacman -Qlq $1 > ./package_file_list
-	echo ".PKGINFO" >> ./package_file_list
-	[[ -f ".INSTALL" ]] && echo ".INSTALL" >> ./package_file_list
-	[[ -f ".Changelog" ]] && echo ".Changelog" >> ./package_file_list
```

Comment:
This change is pretty much independent of the rest of the changes and hence should be in its own commit.
However, I think it does not work as intended as `tar` throws errors if it receives files as input which do not exist. Note, `.INSTALL`, ... must not exist and are only added to the list of files which to back up if they really are present.

Reply:
Yes, I did notice this one. However getting only warning (and only in verbose mode) I did not considered it to be the real issue. I will put the metadata related files into an array so `tar` won't be bothered by nonexisting files.

Reply:
An array is fine, however if there already is a file which contains all the other paths, why not simply amend it like it is done now?

Reply:
I could not find any other way how to put both `$ROOT_PATH/somedir/somefile` and `.PKGINFO` at the top of the archive while at the same time not showing the `$ROOT_PATH` component in the archive.

Reply:
Sorry for the noise, I admittedly did not think of that.